pkg/apis/sources/v1alpha1/cron_job_types.go

```diff
@@ -90,8 +90,7 @@ type CronJobSourceSpec struct {
 	Data string `json:"data,omitempty"`
 
 	// Sink is a reference to an object that will resolve to a domain name to use as the sink.
-	// +optional
-	Sink *corev1.ObjectReference `json:"sink,omitempty"`
+	Sink *apisv1alpha1.Destination `json:"sink,omitempty"`
```

Comment:
This isn't optional, should it still be a pointer?

Reply:
It will get cleaned up in https://github.com/knative/eventing/issues/2111